Update description about the --force parameter (#8206)
pingcap/docs: upgrade-tidb-using-tiup.md

>
> + Performing a rolling upgrade to the cluster will upgrade all components one by one. During the upgrade of TiKV, all leaders in a TiKV instance are evicted before stopping the instance. The default timeout time is 5 minutes (300 seconds). The instance is directly stopped after this timeout time.
>

> > + You can use the `--force` parameter to upgrade the cluster immediately without evicting the leader. However, errors arising in the upgrade will be ignored, which means you are not notified of any upgrade failure. Therefore, use the `--force` parameter with caution.

Reviewer: ```suggestion
> + You can use the `--force` parameter to upgrade the cluster immediately without evicting the leader. However, the errors that occur during the upgrade will be ignored, which means that you are not notified of any upgrade failure. Therefore, use the `--force` parameter with caution.
```
